[[ Bug 20288 ]] Copy CEF process on Win

montegoulding opened the pull request:
This patch changes slash to backslash in the copies block of for
libbrowser-cefprocess.exe. It is unclear why slash does not work
for this target because it does work for revbrowser-cefprocess.exe.

livecodeali commented:
The [develop build succeeds](https://travis-ci.org/livecode/livecode/jobs/308623784), which is super-odd as this definitely appears to be down to the Travis changes.

livecodeali commented:
OK, I think it must be something in this PR somehow. I did a PR with a trivial change and it passes:
https://github.com/livecode/livecode/pull/6179

montegoulding commented:
Hmm... Ok I will see what I can find

montegoulding committed:
[[ Bug 20288 ]] Copy CEF process on Win

This patch changes slash to backslash in the copies block of for
libbrowser-cefprocess.exe. It is unclear why slash does not work
for this target because it does work for revbrowser-cefprocess.exe.

livecodeali commented:
@livecode-vulcan review ok 17f28f1

livecode-vulcan commented:
💙 review by @livecodeali ok 17f28f1b18146b6a8a4b146bb85fce121bd1926d

livecode-vulcan commented:
😎 test success 17f28f1b18146b6a8a4b146bb85fce121bd1926d

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/1018)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1034)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1008)
* try-community-universal-ios-iphoneos11.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.1/builds/81)
* try-community-universal-ios-iphonesimulator11.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.1/builds/81)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/1013)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1035)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/1018)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1034)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/1017)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1027)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1050)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1013)
